crates/ruff_python_parser/src/parser/expression.rs

```diff
@@ -2541,7 +2544,7 @@ impl<'src> Parser<'src> {
             }
         }
 
-        if arguments.len() > 1 {
+        if (has_trailing_comma && arguments.len() == 1) || arguments.len() > 1 {
```

Comment:
I believe we can simplify this to
```suggestion
        if has_trailing_comma || arguments.len() > 1 {
```

since we'll either have 0 arguments, in which case the loop is never entered and the conditional doesn't matter; 1 argument in which case only `has_trailing_comma` matters; or many arguments, in which case `arguments.len() > 1` applies.